public/js/resolve-breaches.js

```diff
@@ -32,7 +32,12 @@
 
   function sendBreachDetailAnalyticsPing(eventCategory, eventAction, eventLabel) {
     if (typeof(ga) !== "undefined") {
-      ga("send", "event", eventCategory, eventAction, eventLabel);
+      // Set view pings as nonInteraction:true to get accurate bounce rate
+      let options = {};
+      if (eventAction !== "Engage"){
+        options = {nonInteraction: true};
```

Comment:
Same as on `fx-analytics.js` – We are wanting to make sure only user-interacted events get marked as interactive in GA.